src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/models/RemoteController.java

```diff
@@ -0,0 +1,157 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.models;
+
+import jakarta.persistence.*;
+
+@Entity
+public class RemoteController {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String type;
+    @Column(nullable = false)
+    private String brand;
+    @Column(nullable = false)
+    private String name;
+    @Column(nullable = false)
+    private Double price;
+    private Boolean smartTv;
+    private Boolean voiceControl;
+    private Boolean bluetooth;
+    private Integer originalStock;
+    private Integer sold;
+
+
+    //Remote controller//
+    private String batteryType;
+    private String compatibleWith;
+
+
+
+
+    public RemoteController() {
+
+    }
+
+
+    public RemoteController(Long id, String type, String brand, String name, Double price, Boolean smartTv, Boolean voiceControl, Boolean bluetooth, Integer originalStock, Integer sold, String batteryType, String compatibleWith){
```

Comment:
Een hele grote constructor. Dit is niet ideaal. Later krijgen we als het goed is nog uitleg over de builder pattern. Die zou je kunnen toepassen evt.

Maar het wat leesbaarder.

Evt zou je ook Lombok kunnen gebruiken en met @data allesgetters en setter plus constructor kunnen maken.

Het is niet slecht is meer een toevoeging op je code.